Source/rclUE/Public/Srvs/ROS2SpawnEntitiesSrv.h

```diff
@@ -0,0 +1,271 @@
+// Copyright 2021 Rapyuta Robotics Co., Ltd.
+// This code has been autogenerated from UE_msgs/SpawnEntities.srv - do not modify
+
+#pragma once
+
+// UE
+#include <CoreMinimal.h>
+
+// rclUE
+#include "Srvs/ROS2GenericSrv.h"
+#include "rclcUtilities.h"
+#include "ue_msgs/srv/spawn_entities.h"
+
+#include "ROS2SpawnEntitiesSrv.generated.h"
+
+// potential problem: if this struct is defined multiple times!
+USTRUCT(Blueprintable)
+struct RCLUE_API FROSSpawnEntitiesRequest
+{
+    GENERATED_BODY()
+
+public:
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FString> TypeList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FString> NameList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FVector> PositionList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FQuat> OrientationList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FVector> TwistLinearList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FVector> TwistAngularList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FString> ReferenceFrameList;
+
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    TArray<FString> TagsList;
+
+    void SetFromROS2(const ue_msgs__srv__SpawnEntities_Request& in_ros_data)
+    {
+        for (int i = 0; i < in_ros_data.type.size; i++)
```

Comment:
```suggestion
        for (auto i = 0; i < in_ros_data.type.size; ++i)
```
as configured [here](https://github.com/rapyuta-robotics/UE_tools/blob/devel/CodeGen/CodeGeneratorFromPath.py#L151)